src/module.js

```diff
@@ -524,21 +524,27 @@ class DiscretePanelCtrl extends CanvasPanelCtrl {
       _.reverse(items);
     }
     
+    var singleMode = (items.length == 1);
```

Comment:
It's a bit strange. 
Single mode should be then it is enabled in options. No?

Reply:
@jonyrock can you please suggest method to get options? Unfortunately, I couldn't find it.